zkevm-circuits/src/evm_circuit/execution/error_oog_log.rs

```diff
@@ -194,34 +228,35 @@ mod test {
         CircuitTestBuilder::new_from_test_ctx(ctx).run();
     }
 
-    fn mock_tx(value: Word, gas_price: Word, calldata: Vec<u8>) -> eth_types::Transaction {
-        let from = MOCK_ACCOUNTS[1];
-        let to = MOCK_ACCOUNTS[0];
-        eth_types::Transaction {
-            from,
-            to: Some(to),
-            value,
-            gas: gas(&calldata),
-            gas_price: Some(gas_price),
-            input: calldata.into(),
-            ..Default::default()
-        }
-    }
-
-    #[test]
-    // test oog log in root call
-    fn test_oog_log_root() {
-        test_oog_log(mock_tx(eth(1), gwei(2), vec![]));
-    }
+    fn test_internal(caller: Account, callee: Account) {
+        let ctx = TestContext::<3, 1>::new(
+            None,
+            |accs| {
+                accs[0]
+                    .address(address!("0x000000000000000000000000000000000000cafe"))
+                    .balance(Word::from(10u64.pow(19)));
+                accs[1]
+                    .address(caller.address)
+                    .code(caller.code)
+                    .nonce(caller.nonce.as_u64())
+                    .balance(caller.balance);
+                accs[2]
+                    .address(callee.address)
+                    .code(callee.code)
+                    .nonce(callee.nonce.as_u64())
+                    .balance(callee.balance);
```

Comment:
```suggestion
                accs[1].account(caller);
                accs[2].account(callee);
```

Reply:
updated in [2236916](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1564/commits/22369163147c2f20c622f8521a59b1ef24e13352) thanks !